taskmanager/src/Application/Task/Task.php

```diff
@@ -21,6 +25,8 @@ public function __construct(string $name, Status $status)
         $this->id = Uuid::uuid4();
         $this->name = $name;
         $this->status = $status;
+        $this->createdAt = new \DateTime();
+        $this->setUpdatedAt();
```

Comment:
maybe sth like `touch(): void`

Reply:
@partikus I wasn't sure about `touch` maybe just `update`? Changed in this way. But if you like `touch` I will touch this PR once more :)

Reply:
that's fine 👍